src/spglib.h

```diff
@@ -184,9 +184,12 @@ typedef struct {
     int type;
 } SpglibMagneticSpacegroupType;
 
-int spg_get_major_version(void);
-int spg_get_minor_version(void);
-int spg_get_micro_version(void);
+const char *spg_get_version();
+const char *spg_get_version_full();
+const char *spg_get_commit();
```

Comment:
Can you update ChangeLog.md to introduce these new functions?

Reply:
Sure, but first, any suggestions about the naming of the functions?

Reply:
I think these three namings are fine. @atztogo Do you have any preference for these functions?

Reply:
ping @atztogo

Reply:
I am assuming @atztogo gave the :+1: for this?

Reply:
Thanks for pings!